classes.fields.php

```diff
@@ -213,6 +222,47 @@ public function boolean_attr( $attrs = array() ) {
 	}
 
 	/**
+	 * Check for validation array key in CMB Field.
+	 *
+	 * @return boolean
+	 */
+	public function should_validate_values() {
+		return ( ! empty( $this->args['validation'] ) );
+	}
+
+	/**
+	 * Return escaped parsley attributes
+	 * supports: required, required => minlength, required => maxlength
+	 *
+	 * @param array $attrs
+	 *
+	 * @todo Support all Parsley default validation types.
+	 */
+	public function validation_attr( $attrs = array() ) {
+
+		if ( ! $this->should_validate_values() ) {
+			return;
+		}
+
+		$validation_rules = wp_parse_args( $attrs, $this->args['validation'] );
+
+		// Validation rule: Required.
+		if ( isset( $validation_rules['required'] ) ) {
+
+			echo esc_attr( ' required ' );
+
+			if ( is_array( $validation_rules['required'] ) ) {
```

Comment:
this feels a little odd to me... do you pass the args as an array on the 'required' property of the validation array?

I'd prefer the config to be in this format:

```
'validation' => [
    'required'   => true,
    'minlength' => 10, 
]
```

Then output attributes something like this...

```
foreach ( $validation_rules as $attr => $value ) {
    if ( 'required' === $attr && $attr ) {
        echo ' required';
    if ( is_bool( $value ) ) {
        printf( ' data-parseley-%s', esc_attr( $attr ) );
    } else {
        printf( ' data-parseley-%s="%s"', esc_attr( $attr ) );
    }
}
```